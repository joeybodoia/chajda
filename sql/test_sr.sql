CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."');
