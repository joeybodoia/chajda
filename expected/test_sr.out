CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                           spacy_lemmatize                                           
-----------------------------------------------------------------------------------------------------
   пример:5 тестни:6 реченице:7 стављати:9 гоогле:11 транслате:12 генерисати:15 тест:16 случајеве:17
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                               spacy_lemmatize                                
------------------------------------------------------------------------------
   пример тестни реченице стављати гоогле транслате генерисати тест случајеве
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                                   spacy_lemmatize                                                                   
-----------------------------------------------------------------------------------------------------------------------------------------------------
   овај:2 бити:3 мој:4 пример:5 тестни:6 реченице:7 који:8 стављати:9 у:10 гоогле:11 транслате:12 дати:13 бити:14 генерисати:15 тест:16 случајеве:17
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                               spacy_lemmatize                                               
-------------------------------------------------------------------------------------------------------------
   овај бити мој пример тестни реченице који стављати у гоогле транслате дати бити генерисати тест случајеве
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                  spacy_lemmatize                                                   
--------------------------------------------------------------------------------------------------------------------
   ":2 пример:6 тестни:7 реченице:8 стављати:10 гоогле:12 транслате:13 генерисати:16 тест:17 случајеве:18 .:19 ":20
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                  spacy_lemmatize                                   
------------------------------------------------------------------------------------
   " пример тестни реченице стављати гоогле транслате генерисати тест случајеве . "
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                          spacy_lemmatize                                                                           
--------------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 овај:3 бити:4 мој:5 пример:6 тестни:7 реченице:8 који:9 стављати:10 у:11 гоогле:12 транслате:13 дати:14 бити:15 генерисати:16 тест:17 случајеве:18 .:19 ":20
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                                  spacy_lemmatize                                                  
-------------------------------------------------------------------------------------------------------------------
   " овај бити мој пример тестни реченице који стављати у гоогле транслате дати бити генерисати тест случајеве . "
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                           spacy_lemmatize                                           
-----------------------------------------------------------------------------------------------------
   пример:5 тестни:6 реченице:7 стављати:9 Гоогле:11 Транслате:12 генерисати:15 тест:16 случајеве:17
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                               spacy_lemmatize                                
------------------------------------------------------------------------------
   пример тестни реченице стављати Гоогле Транслате генерисати тест случајеве
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                                  spacy_lemmatize                                                                   
----------------------------------------------------------------------------------------------------------------------------------------------------
   Ово:2 бити:3 мој:4 пример:5 тестни:6 реченице:7 који:8 стављати:9 у:10 Гоогле:11 Транслате:12 дати:13 бити:14 генерисати:15 тест:16 случајеве:17
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                              spacy_lemmatize                                               
------------------------------------------------------------------------------------------------------------
   Ово бити мој пример тестни реченице који стављати у Гоогле Транслате дати бити генерисати тест случајеве
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                  spacy_lemmatize                                                   
--------------------------------------------------------------------------------------------------------------------
   ":2 пример:6 тестни:7 реченице:8 стављати:10 Гоогле:12 Транслате:13 генерисати:16 тест:17 случајеве:18 .:19 ":20
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                  spacy_lemmatize                                   
------------------------------------------------------------------------------------
   " пример тестни реченице стављати Гоогле Транслате генерисати тест случајеве . "
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                          spacy_lemmatize                                                                          
-------------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 Ово:3 бити:4 мој:5 пример:6 тестни:7 реченице:8 који:9 стављати:10 у:11 Гоогле:12 Транслате:13 дати:14 бити:15 генерисати:16 тест:17 случајеве:18 .:19 ":20
(1 row)

SELECT spacy_lemmatize('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                                 spacy_lemmatize                                                  
------------------------------------------------------------------------------------------------------------------
   " Ово бити мој пример тестни реченице који стављати у Гоогле Транслате дати бити генерисати тест случајеве . "
(1 row)

SELECT spacy_tsvector('sr',' "Ово је мој пример тестне реченице коју стављам у Гоогле Транслате да бих генерисао тест случајеве."');
                                                   spacy_tsvector                                                    
---------------------------------------------------------------------------------------------------------------------
 'генерисати':15 'гоогле':11 'пример':5 'реченице':7 'случајеве':17 'стављати':9 'тест':16 'тестни':6 'транслате':12
(1 row)

